android/java/org/chromium/chrome/browser/app/bookmarks/BraveBookmarkActivity.java

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.app.bookmarks;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.chromium.chrome.browser.bookmarks.BookmarkManager;
+import org.chromium.chrome.browser.bookmarks.BraveBookmarkManager;
+import org.chromium.ui.base.ActivityWindowAndroid;
+import org.chromium.ui.base.IntentRequestTracker;
+
+public class BraveBookmarkActivity extends BookmarkActivity {
+    private BookmarkManager mBookmarkManager;
+    private ActivityWindowAndroid mWindowAndroid;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        IntentRequestTracker intentRequestTracker = IntentRequestTracker.createFromActivity(this);
+        mWindowAndroid = new ActivityWindowAndroid(this, true, intentRequestTracker);
+        mWindowAndroid.getIntentRequestTracker().restoreInstanceState(savedInstanceState);
+        if (mBookmarkManager != null && mBookmarkManager instanceof BraveBookmarkManager) {
```

Comment:
I don't understand, where `mBookmarkManager` is set? It is used but never assigned.

Reply:
Ah, sorry, it is from asm patch at
https://github.com/brave/brave-core/pull/17821/files#diff-7c55d46804cdd04b2228e0884363ba27cbd181473caccf0bfd62052c496e11bbR19

Could you please add comment above `private BookmarkManager mBookmarkManager` that it is 
`// Overridden Chromium's BookmarkActivity.mBookmarkManager` ?